src/primitives/certificate.cpp

```diff
@@ -64,6 +64,16 @@ bool CScCertificate::CheckVersionBasic(CValidationState &state) const
     return true;
 }
 
+bool CScCertificate::CheckVersionIsStandard(std::string& reason, const int nHeight) const {
```

Comment:
Here is the right place for adding certificate version checking, that was missing even in original code

Reply:
Agree. I think also CheckVersionBasic may be filled with a gross check about min max version as with transactions. Also a ContextualVersionCheck should be added for certs, which now is missing.
I would like to fix the version checks in one session rather than here, however. Here I'd content myself with opening up the places for checks